live/README.md

```diff
@@ -0,0 +1,38 @@
+# Building live cd
+
+ * cd live
+ * build the rootfs
+ 
+    ```
+    stacker build --layer-type=squashfs
+    ```
+
+   * Note: I  seem to be hitting a bug with this, where it fails to build the second time.
+
+ * get privkey.pem and cert.pem and run 'build-media'
+
+   ```
+   ./build-media \
+        --cert=/path/to/keys/manifest/cert.pem  \
+        --key=/path/to/keys/manifest/privkey.pem \
+        out.img \
+        docker://zothub.io/machine/bootkit/bootkit:0.0.5.230327-squashfs
+        oci:./oci:rootfs-squashfs
+    ```
+
+ * boot the usb media. 
+ 
+   * atomix-vm-builder wants qcow2 format, so convert to qcow2
+   * build-media left 'ovmf-vars.fd' and 'ovmf-code.fd' in same dir as out.img
+
+    ```
+    rm -f out.qcow2 
+    qemu-img create -fqcow2 -b out.img -Fraw out.qcow2 
+    atomix-vm-builder run \
```

Comment:
Is there anything that this does that mcli won't do?

Reply:
as you found, i guess usb. but that isnt' necessary, just needs to be attached somehow that can be booted.